common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -862,10 +867,11 @@ def test_full_pipeline_succeeds_registering_new_account(self):
         # Set the cookie and try again
         self.set_logged_in_cookies(request)
         self.assert_redirect_to_dashboard_looks_correct(
-            actions.do_complete(strategy.request.backend, social_views._do_login, user=created_user))
+            actions.do_complete(strategy.request.backend, social_views._do_login, user=created_user)
+        )
```

Comment:
As above - we don't need to change this, so we shouldn't fix the style. Clean diffs are important; they allow us to see the actual code changes much more easily.